conn.go

```diff
@@ -1356,28 +1356,35 @@ func (c *Conn) query(ctx context.Context, statement string, values ...interface{
 
 func (c *Conn) awaitSchemaAgreement(ctx context.Context) (err error) {
 	const (
-		peerSchemas  = "SELECT schema_version, peer FROM system.peers"
+		peerSchemas  = "SELECT * FROM system.peers"
 		localSchemas = "SELECT schema_version FROM system.local WHERE key='local'"
 	)
 
 	var versions map[string]struct{}
+	var schemaVersion string
 
 	endDeadline := time.Now().Add(c.session.cfg.MaxWaitSchemaAgreement)
 	for time.Now().Before(endDeadline) {
 		iter := c.query(ctx, peerSchemas)
 
 		versions = make(map[string]struct{})
 
-		var schemaVersion string
-		var peer string
-		for iter.Scan(&schemaVersion, &peer) {
-			if schemaVersion == "" {
-				Logger.Printf("skipping peer entry with empty schema_version: peer=%q", peer)
+		rows, err := iter.SliceMap()
+		if err != nil {
+			goto cont
+		}
+
+		for _, row := range rows {
+			host, err := c.session.hostInfoFromMap(row, c.session.cfg.Port)
```

Comment:
We're getting error at this line. Please advice.
cannot use c.session.cfg.Port (type int) as type *HostInfo in argument to c.session.hostInfoFromMap

Reply:
Yep! delayed our releases because of this ... seems CI failed on master!

What about making releases so that you won't push broken code to master?

Reply:
Sorry about that. All tests passed fine, so we just missed this one. Anyway, a fix is on the way.